packages/svelte/package.json

```diff
@@ -2,67 +2,97 @@
   "name": "@ai-sdk/svelte",
   "version": "1.1.19",
   "license": "Apache-2.0",
-  "sideEffects": false,
-  "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
   "scripts": {
-    "build": "tsup",
-    "build:watch": "tsup --watch",
-    "clean": "rm -rf dist",
-    "lint": "eslint \"./**/*.ts*\"",
-    "type-check": "tsc --noEmit",
-    "prettier-check": "prettier --check \"./**/*.ts*\""
+    "dev": "vite dev",
+    "build": "vite build && npm run prepack",
+    "preview": "vite preview",
+    "prepare": "svelte-kit sync || echo ''",
+    "prepack": "svelte-kit sync && svelte-package && publint",
+    "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+    "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
+    "format": "prettier --write .",
+    "lint": "prettier --check . && eslint .",
+    "test": "vitest --run",
+    "test:watch": "vitest"
   },
+  "files": [
+    "dist",
+    "!dist/**/*.test.*",
+    "!dist/**/*.spec.*",
+    "src/lib",
+    "!src/lib/**/*.test.*",
+    "!src/lib/**/*.spec.*",
+    "CHANGELOG.md"
+  ],
+  "sideEffects": [
+    "**/*.css"
+  ],
+  "svelte": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "type": "module",
   "exports": {
-    "./package.json": "./package.json",
     ".": {
       "types": "./dist/index.d.ts",
-      "import": "./dist/index.mjs",
-      "require": "./dist/index.js"
+      "svelte": "./dist/index.js"
     }
   },
-  "files": [
-    "dist/**/*",
-    "CHANGELOG.md"
-  ],
-  "dependencies": {
-    "@ai-sdk/provider-utils": "2.1.10",
-    "@ai-sdk/ui-utils": "1.1.16",
-    "sswr": "^2.1.0"
-  },
-  "devDependencies": {
-    "@types/node": "^18",
-    "@vercel/ai-tsconfig": "workspace:*",
-    "eslint": "8.57.1",
-    "eslint-config-vercel-ai": "workspace:*",
-    "tsup": "^7.2.0",
-    "typescript": "5.6.3"
+  "publishConfig": {
+    "access": "public"
   },
   "peerDependencies": {
-    "svelte": "^3.0.0 || ^4.0.0 || ^5.0.0"
-  },
-  "peerDependenciesMeta": {
-    "svelte": {
-      "optional": true
-    }
-  },
-  "engines": {
-    "node": ">=18"
+    "svelte": "^5.0.0"
   },
-  "publishConfig": {
-    "access": "public"
+  "devDependencies": {
+    "@ai-sdk/openai": "1.2.0",
```

Comment:
There are a lot more dev dependencies so that the package can include the full set of hook demos -- makes local development a lot easier